[2024-04-13 20:16:00] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:05] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:10] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:15] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:20] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:22] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 905d54621bd8d9e7 [::1]:45152 failed checkpoint posthandshake: Connected to self
[2024-04-13 20:16:22] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 905d54621bd8d9e7 [::1]:20136 failed checkpoint posthandshake: Connected to self
[2024-04-13 20:16:25] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:30] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:35] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:40] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:45] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:50] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:55] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:16:55] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 905d54621bd8d9e7 [::1]:44594 failed checkpoint posthandshake: Connected to self
[2024-04-13 20:16:55] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 905d54621bd8d9e7 [::1]:20136 failed checkpoint posthandshake: Connected to self
